app/src/main/java/com/dessalines/thumbkey/db/AppDb.kt

```diff
@@ -298,6 +304,14 @@ val MIGRATION_9_10 = object : Migration(9, 10) {
     }
 }
 
+val MIGRATION_10_11 = object : Migration(10, 11) {
+    override fun migrate(database: SupportSQLiteDatabase) {
+        database.execSQL(
+            "alter table AppSettings add column backdrop_enabled INTEGER NOT NULL default $DEFAULT_BACKDROP_ENABLED",
+        )
+    }
+}
+
```

Comment:
You also need to update the database version in line 316 to 11.

Reply:
I must have done something funny because this was working when I tested on the virtual device, but I've figured out how to test on my phone now so hopefully that's more reliable.